_includes/community/meshmates-program-summary.html

```diff
@@ -1,26 +1,54 @@
-  <p class="indent">
-    The Layer5 community is growing at a tremendous rate. No matter the size, we know that connection is what makes the Layer5 community different from any other community you join. Whether it is your first Layer5 meeting or your tenth contribution, as community organizers, we want to make sure you have a great time and are able to take advantage of all of the 
-    amazing learning, contributing, and community that Layer5 has to offer. 
-  </p>  
-  <p class="indent">
-    In the Layer5 mentor program, we will match you with an 
-    awesome MeshMate so you have a connection and guide from the start.
-  </p>
+<table>
+<tr>
+  <th>Group</th>
+  <th>Slack Channel</th>
+  <th>Meeting Minutes</th> 
+  <th>Meeting Link</th>
+  <th>Meeting Recordings</th>
+</tr>
+<tr>
+  <td>Layer5 Community Meeting</td>
+  <td><a href = "https://layer5io.slack.com/archives/CDM0ACDM5">#general</a></td>
+  <td><a href="https://bit.ly/2It7IA4">Minutes</a></td>
+  <td><a href="https://bit.ly/33otH4x">Zoom Link</a></td>
+  <td><a href = "https://www.youtube.com/playlist?list=PL3A-A6hPO2IMPPqVjuzgqNU5xwnFFn3n0">Youtube</a></td>
+</tr>
+<tr>
+  <td>Websites Meeting</td>
+  <td><a href = "https://layer5io.slack.com/archives/C015QJKUMPU">#websites</a></td>
+  <td><a href = "https://bit.ly/2XK4eQV"> Minutes</a></td>
+  <td><a href = "https://zoom.us/s/96393160667">Zoom Link</a></td>
+  <td><a href = "https://www.youtube.com/watch?v=Zba0b0ZojOI&list=PL3A-A6hPO2IO1ZuLj3cbOTdi-hlkF4Lqu">Youtube</a></td>
+</tr>
+<tr>
+  <td>Meshery Development Meeting</td>
+  <td><a href = "https://layer5io.slack.com/archives/CFGG6U10E">#meshery</a></td>
+  <td><a href = "https://bit.ly/31OGQTk">Minutes</a></td>
+  <td><a href = "https://bit.ly/2FpLctk">Zoom Link</a></td>
+  <td><a href = "https://www.youtube.com/watch?v=SJctAfZ52d8&list=PL3A-A6hPO2INr4oEIorJqC8N0FtxnYj0a">Youtube</a></td>
+</tr>
+<tr>
+  <td>MeshSync Meeting</td>
+  <td><a href = "https://layer5io.slack.com/archives/C011VPREG2Z">#meshery-ci</a></td>
+  <td><a href = "https://bit.ly/3aXOaAs ">Minutes</a></td>
+  <td><a href = "https://bit.ly/2QocW3s">Zoom Link</a></td>
+  <td><a href = "https://www.youtube.com/watch?v=ds9D2KgZKxo&list=PL3A-A6hPO2IM7rYiKxG4l3eQNc6X3IUex">Youtube</a></td>
+</tr>
+<tr>
+  <td>Newcomers' Meeting</td>
+  <td><a href = "https://layer5io.slack.com/archives/C019426UBNY">#newcomers</a></td>
+  <td><a href = "https://bit.ly/353Aa7D">Minutes</a></td>
+  <td><a href = "https://bit.ly/34SFIlc">Zoom Link</a></td>
+  <td><a href = "https://www.youtube.com/watch?v=E1juH6Rjovo&list=PL3A-A6hPO2INVEknQ4R-VwDMUT9yM_FSm">Youtube</a></td>
+</tr>
+<tr>
+  <td>SMI Community Meeting</td>
+  <td><a href = "https://layer5io.slack.com/archives/C010H0HE2E6">#smi</a></td>
+  <td><a href ="https://docs.google.com/document/d/1NTBaJf6LhUBlF8_lfvBBt_MbyPvT-6CZNg6Ckpm_yCo/edit">Minutes</a></td>
+  <td><a href = "https://zoom.us/my/cncfsmiproject"> Zoom Link </a></td>
```

Comment:
If we're going to list the SMI Community Meeting, let's go ahead and list the CNCF SIG Network meeting as well.

Both of these two meetings should be colorized or marked differently somehow to show that both of them are CNCF meetings.